tap.go

```diff
@@ -1,24 +1,34 @@
 package tap
 
+import "io"
+import "os"
 import "fmt"
 import "testing/quick"
 
 type T struct {
 	nextTestNumber int
+	outputFile     *os.File
 }
 
 // New creates a new Tap value
-func New() *T {
+func New(fileName string) *T {
+	output, err := os.Create(fileName)
+	if err != nil {
+		panic(err)
```

Comment:
It's usually best to return the error value rather than panic.  That way the caller can decide how to handle the error.